.github/workflows/Pipeline.yml

```diff
@@ -1,450 +1,116 @@
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ env.PYTHON }}
-
-      - name: 🗂 Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r tests/requirements.txt
-
-      - name: Check Static Typing
-        continue-on-error: true
-        run: |
-          pwd
-          mypy --html-report htmlmypy -m pyEDAA.ProjectModel
-
-      - name: 📤 Upload 'Static Typing Report' artifact
-        continue-on-error: true
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ env.ARTIFACT }}
-          path: htmlmypy
-          if-no-files-found: error
-          retention-days: 1
-
+    uses: pyTooling/Actions/.github/workflows/StaticTypeCheck.yml@v0
+    needs:
+      - Params
+    with:
+      python_version: ${{ fromJson(needs.Params.outputs.params).python_version }}
+      commands: |
+        cd pyEDAA
+        mypy --html-report ../htmlmypy -p ProjectModel
+      artifact: ${{ fromJson(needs.Params.outputs.params).artifacts.typing }}
 
   Release:
-    name: 📝 Create 'Release Page' on GitHub
-    runs-on: ubuntu-latest
-
+    uses: pyTooling/Actions/.github/workflows/Release.yml@v0
     if: startsWith(github.ref, 'refs/tags')
     needs:
       - UnitTesting
       - Coverage
       - StaticTypeCheck
 
-    env:
-      PYTHON: ${{ needs.Coverage.outputs.python }}
-    outputs:
-      python:     ${{ env.PYTHON }}
-      tag:        ${{ steps.getVariables.outputs.gitTag }}
-      version:    ${{ steps.getVariables.outputs.version }}
-      datetime:   ${{ steps.getVariables.outputs.datetime }}
-      upload_url: ${{ steps.createReleasePage.outputs.upload_url }}
-
-    steps:
-      - name: 🔁 Extract Git tag from GITHUB_REF
-        id:   getVariables
-        run: |
-          GIT_TAG=${GITHUB_REF#refs/*/}
-          RELEASE_VERSION=${GIT_TAG#v}
-          RELEASE_DATETIME="$(date --utc '+%d.%m.%Y - %H:%M:%S')"
-          # write to step outputs
-          echo ::set-output name=gitTag::${GIT_TAG}
-          echo ::set-output name=version::${RELEASE_VERSION}
-          echo ::set-output name=datetime::${RELEASE_DATETIME}
-
-      - name: 📑 Create Release Page
-        id:   createReleasePage
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ steps.getVariables.outputs.gitTag }}
-#          release_name: ${{ steps.getVariables.outputs.gitTag }}
-          body: |
-            **Automated Release created on: ${{ steps.getVariables.outputs.datetime }}**
-
-            # New Features
-            * tbd
-
-            # Changes
-            * tbd
-
-            # Bug Fixes
-            * tbd
-          draft: false
-          prerelease: false
-
   Package:
-    name: 📦 Package in Wheel Format
-    runs-on: ubuntu-latest
-
+    uses: pyTooling/Actions/.github/workflows/Package.yml@v0
     if: startsWith(github.ref, 'refs/tags')
```

Comment:
In latest workflows like pyTooling, this step is not conditional anymore. I saw lots of problems when the wheel was generated. So I would like to see it packaged every time. (or can we limit to if a PR is opened?)

So wheel files are always generated.

Reply:
This is now fixed, both here and in the ExamplePipeline of pyTooling/Actions.